packages/dev-app/src/data/search-response.json

```diff
@@ -0,0 +1,593 @@
s fluctuant, or they become systemically unwell. Specialist advice should be sought if there is a possibility of PVL-SA. A course of antibiotics (flucloxacillin first-line; erythromycin or clarithromycin if the person is allergic to penicillin) should be prescribed if\u0026nbsp;there is cellulitis, fever, a facial lesion or severe pain; a carbuncle is present; or there are other comorbidities, such as diabetes or immunosuppression. Self-care advice should be offered.   If the boils or carbuncles are recurrent, the lesion should be swabbed and the person treated with appropriate antibiotics for 7 days. If PVL-SA infection is suspected, this should be specifically mentioned on the laboratory form. If PVL-SA is confirmed, management should be discussed with microbiology. Staphylococcal carriage (colonization) refers to the asymptomatic carriage of\u0026nbsp;S. aureus\u0026nbsp; on a person\u0026#39;s skin or mucous membranes.The most common site of colonization by\u0026nbsp;S. aureus\u0026nbsp; is the nose.\u0026nbsp;   Staphylococcal carriage is a risk factor for recurrent boils and carbuncles.\u0026nbsp; Swabs of the nose should be taken to test for staphylococcal carriage if recurrent boils are in the facial area. If the recurrent boils are more extensive, swabs should also be taken from the perineum, groin, axilla and umbilicus.\u0026nbsp; If staphylococcal carriage is confirmed, the person should be prescribed nasal decolonization (with naseptin unless the person is allergic to peanuts or soya) and advised on skin treatment. Decolonization should not be started until the acute infection has resolved.\u0026nbsp; If there is a recurrence of boils or carbuncles despite treating staphylococcal carriage, management includes:Consideration of extending swabbing to exclude carriage at other sites such as the perineum, groin, axillae, and umbilicus. Specifically requesting testing for PVL-SA. Identifying and treating potential sources of infection in the family and close contacts.\",\"feedertype\":\"pfeeders\",\"published_date\":\"2017-01-01T12:00:00\",\"source_name\":\"CKS\"},\"highlight\":{\"content\":[\"folliculitis, cellulitis or cutaneous abscess. There are separate CKS topics on \u003cmark\u003eAcne\u003c/mark\u003e vulgaris , Candida - skin , Cellulitis - acute , Fungal nail infection , Fungal\",\"  \\n    Conditions which may resemble a boil or carbuncle include: Cystic \u003cmark\u003eacne\u003c/mark\u003e  — associated with nodules, papules, and comedones and usually confined to\",\"epidermis and contains sebum and keratin. These are frequently seen in severe \u003cmark\u003eacne.\u003c/mark\u003e They may become inflamed and tender intermittently and discharge pus. Folliculitis \",\"including atypical mycobacteria infections [Yates and Rook, 2004 ], cystic \u003cmark\u003eacne\u003c/mark\u003e [Craft, 2012 ], dental abscess and osteomyelitis [Craft, 2012 ], epidermoid\"]}}]},\"aggregations\":{\"cardinality#unique_count\":{\"value\":25}}}\r\n"
```

Comment:
Can we remove the debug property please? It's a bit of noise - we just need the properties like resultcount